StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/PaymentSheetFormFactory.swift

```diff
@@ -697,10 +698,7 @@ extension PaymentSheetFormFactory {
     }
 
     func makeKlarnaCopyLabel() -> StaticElement {
-        let text =
-            KlarnaHelper.canBuyNow()
-            ? STPLocalizedString("Buy now or pay later with Klarna.", "Klarna buy now or pay later copy")
-            : STPLocalizedString("Pay later with Klarna.", "Klarna pay later copy")
+        let text = STPLocalizedString("Buy now or pay later with Klarna.", "Klarna buy now or pay later copy")
```

Comment:
Pretty much all countries now support "Buy Now", only 1 deoesn't which is New Zealand. I think it's safe to remove this client side logic now.

Reply:
Confirmed web doesn't have any "Buy Now" only messaging. cc @fabianlindfors-stripe